export_gpg_public

```diff
@@ -15,6 +15,13 @@ fi
 
 CHECKOUT="${GIT_DIR}/theforeman.org"
 
+if [[ ! -d "$CHECKOUT" ]] ; then
+	git clone --quiet --origin "$GIT_REMOTE" https://github.com/theforeman/theforeman.org "$CHECKOUT"
+fi
+
+git -C "$CHECKOUT" fetch "$GIT_REMOTE"
+git -C "$CHECKOUT" checkout -b "add-gpg-key-$VERSION" "$GIT_REMOTE/$TFM_WEBSITE_GH_PAGES_BRANCH"
```

Comment:
Is this variable defined? On my phone so annoying to check but I don't recall having this tfm var

Reply:
Great catch! I forgot to commit my changes in the _settings_ file. Fixed in the latest version.